rust/actyx/ax/src/cmd/settings/mod.rs

```diff
@@ -33,6 +38,7 @@ pub fn run(opts: SettingsOpts, json: bool) -> Box<dyn Future<Output = ()> + Unpi
         SettingsOpts::Get(opt) => get::SettingsGet::output(opt, json),
         SettingsOpts::Schema(opt) => schema::SettingsSchema::output(opt, json),
         SettingsOpts::Unset(opt) => unset::SettingsUnset::output(opt, json),
+        SettingsOpts::Local(opt) => local::run(opt, json),
```

Comment:
This made me think, should we use `remote` and `local` instead of the "normal" case being `get` and friends?

Reply:
My initial thought as well, but "breaking change"

Reply:
I mean, it's a shift paradigm into `ax` treats local and remote as the same class